history-of-quakers/modernized/22-chapter-20.adoc

```diff
@@ -951,15 +1045,29 @@ not only in word, but in deed also,
 
 I write this at the request of certain of my friends called Quakers.
 
-W+++.+++ SEWEL
+[.signed-section-signature]
+W+++.+++ Sewel
 
+[.signed-section-context-close]
 Amsterdam, the 23rd day of the month called April, 1679.
 
-Some questions presented to the Catholics, and the Pope of Rome,
-as the supreme head of their church, and commended to their consideration by George Fox.
+--
+
+[.embedded-content-document.treatise]
+--
+
+[.small-break]
+'''
+
+[.letter-heading]
+Some questions presented to the Catholics,
+and the Pope of Rome, as the supreme head of their church,
+and commended to their consideration by George Fox.
 
+[.salutation]
 Friends,
 
+[.numbered]
```

Comment:
So this whole treatise looks a little funny in the PDF being numbered with multiple paragraphs and the breaks between the numbered paragraphs and the others after them, if you know what I mean. Not sure if there's anything different to do about it.

Reply:
ok, i thought it looked pretty good in the pdf, so i think i'm comfortable with what you did.

Reply:
okay, sounds good. It's just me then. 😄